packages/sdk/artifacts/IDelegationRegistry.json

```diff
@@ -0,0 +1,501 @@
+[
```

Comment:
Since we need to call `getDelegationsByDelegate` from the SDK, we also need to have typechain generate types for this interface.